src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const listItem = document.querySelectorAll('li');
+const tree = document.querySelector('.tree');
+
+for (const x of listItem) {
+  const span = document.createElement('span');
+
+  if (x.children.length !== 0) {
+    x.prepend(span);
+    span.append(span.nextSibling);
+  }
+}
+
+tree.addEventListener('click', e => {
+  const item = e.target.nextSibling;
```

Comment:
To avoid selecting elements other than 'li' elements with children, check if the target element is a 'span' before accessing the 'nextSibling'. Add this condition: 'if (e.target.tagName === 'SPAN') {'.